tests/sjs-junit/test/org/scalajs/testsuite/compiler/CustomSelectableTestScala3.scala

```diff
@@ -0,0 +1,28 @@
+package org.scalajs.testsuite.compiler
+
+import scala.reflect.Selectable
+
+import org.junit.Assert._
+import org.junit.Test
+
+class CustomSelectableTestScala3 {
```

Comment:
Consider calling this `CustomReflectSelectableTestScala3`. It took me a while to realize why this even worked, because I thought it's a custom `scala.Selectable`.

Alternatively, import `scala.reflect` and write `reflect.Selectable`? (similar to collection styles).

Reply:
Good point! I did both ;)